### PR TITLE
clean up remaining deprecations

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 1]
```diff
@@ -600,10 +600,6 @@ public function mirror($originDir, $targetDir, \Traversable $iterator = null, $o
      */
     public function isAbsolutePath($file)
     {
-        if (null === $file) {
-            @trigger_error(sprintf('Calling "%s()" with a null in the $file argument is deprecated since Symfony 4.4.', __METHOD__), E_USER_DEPRECATED);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Filesystem/Filesystem.php]: Should we throw instead?

Comment (xabbuh) [reply src/Symfony/Component/Filesystem/Filesystem.php]: There is another PR adding a type hint for this argument

Comment (derrabus) [reply src/Symfony/Component/Filesystem/Filesystem.php]: #32247

**Review by Tobion — APPROVED** [review]

**Comment by Tobion** [comment]
Thank you @xabbuh.

**Pull request merged**